zetaclient/chains/bitcoin/observer/utxos.go

```diff
@@ -180,18 +181,27 @@ func (ob *Observer) SelectUTXOs(
 
 // findNonceMarkUTXO finds the nonce-mark UTXO in the list of UTXOs.
 func (ob *Observer) findNonceMarkUTXO(nonce uint64, txid string) (int, error) {
+	logger := ob.logger.Outbound.With().Str(logs.FieldMethod, "findNonceMarkUTXO").Logger()
+
 	tssAddress := ob.TSSAddressString()
 	amount := chains.NonceMarkAmount(nonce)
+
 	for i, utxo := range ob.utxos {
 		sats, err := common.GetSatoshis(utxo.Amount)
 		if err != nil {
-			ob.logger.Outbound.Error().Err(err).Msgf("FindNonceMarkUTXO: error getting satoshis for utxo %v", utxo)
+			logger.Error().
+				Err(err).
+				Any("utxo", utxo).
+				Msg("error getting satoshis for utxo")
 		}
 		if utxo.Address == tssAddress && sats == amount && utxo.TxID == txid && utxo.Vout == 0 {
-			ob.logger.Outbound.Info().
-				Msgf("FindNonceMarkUTXO: found nonce-mark utxo with txid %s, amount %d satoshi", utxo.TxID, sats)
+			logger.Info().
+				Str("utxo_tx_id", utxo.TxID).
+				Int64("satoshis_amount", sats).
```

Comment:
We might just want to use `amount` for this one, it's the base amount just like wei for eth chains